src/components/Header/Header.tsx

```diff
@@ -0,0 +1,22 @@
+import Logo from "../../assets/logo.svg";
+import Cart from "../../assets/cart.svg";
+import { HeaderStyle, CartCount, LogoImg, CartImg } from "./Header.style";
+import { useContext, useEffect, useState } from "react";
+import { CartItemsContext } from "../../context/CartItemsContext";
+
+export default function Header() {
+  const [quantityInCart, setQuantityInCart] = useState(0);
+  const { cartItems } = useContext(CartItemsContext);
+
+  useEffect(() => {
+    setQuantityInCart(cartItems.length);
+  }, [cartItems]);
+
+  return (
+    <HeaderStyle>
+      <LogoImg src={Logo} alt="로고" />
+      <CartImg src={Cart} alt="장바구니" />
+      {quantityInCart && <CartCount>{quantityInCart}</CartCount>}
```

Comment:
아래처럼 사용하는 것과 차이가 있을까요?

```suggestion
      {cartItems?.length && <CartCount>{cartItems.length}</CartCount>}
```

Reply:
오.. 아뇨. 전혀 차이가 없을 것 같습니다!
불필요하게 코드라인을 늘리고 있었네요. 수정하겠습니다!